src/app/options/qgscustomprojectionoptions.cpp

```diff
@@ -420,7 +420,7 @@ QString QgsCustomProjectionOptionsWidget::multiLineWktToSingleLine( const QStrin
 
 QString QgsCustomProjectionOptionsWidget::helpKey() const
 {
-  return QStringLiteral( "working_with_projections/working_with_projections.html" );
+  return QStringLiteral( "introduction/qgis_configuration.html#user-defined-crs" );
```

Comment:
I'm divided here. The https://docs.qgis.org/3.34/en/docs/user_manual/working_with_projections/working_with_projections.html#custom-coordinate-reference-system is somewhat more descriptive (and a bit of duplicate afaics --- we should probably keep only one detailed section)

Reply:
Ok, I agree that that's nicer. I think at least `introduction/qgis_configuration.html#user-defined-crs` should include a link to this extra info!

Reply:
I've reverted it now